test/functional/helpers/report-helper.js

```diff
@@ -0,0 +1,63 @@
+function checkTestRelevanceByBrowserAlias (test, browserAlias) {

```

Comment:
`filterErrorsByUserAgents` is a bit simpler than everything else, so we can go with it (unless `filterErrors` is enough)